pkg/config/config.go

```diff
@@ -319,6 +319,8 @@ func initConfig(config Config) {
 
 	// Trace agent
 	config.BindEnvAndSetDefault("apm_config.enabled", true)
+	config.BindEnvAndSetDefault("apm_config.log_throttling", true)
```

Comment:
I'd rather not move any defaults here yet. This will end up causing difficult to spot bugs because everyone expects the defaults in the trace-agent code to be used. Let's remove this one for now. For now they live in https://github.com/DataDog/datadog-agent/blob/master/pkg/trace/config/config.go#L116-L150

Later on, if this is needed, we can schedule a small refactor to completely move them over here.